core/templates/domain/platform_feature/platform-feature-dummy-backend-api.service.ts

```diff
@@ -47,6 +47,7 @@ export class PlatformFeatureDummyBackendApiService {
       await this.http.get(PlatformFeatureDomainConstants.DUMMY_HANDLER_URL)
         .toPromise();
       return true;
+    // Unknown type is used because we don't know which type of error is thrown.
```

Comment:
Here & elsewhere: this statement would be stronger if it stated that we can't know what type of error is thrown & why. The code already implies that we don't currently know the error, but the reason we can't improve the code to catch a more specific error is the actual piece of information that's important to document in context.

Reply:
Done.